pcapps/pythd.cpp

```diff
@@ -112,6 +113,7 @@ int main(int argc, char **argv)
       case 'w': cnt_dir = optarg; break;
       case 'l': log_file = optarg; break;
       case 'm': cmt = str_to_commitment(optarg); break;
+      case 'b': max_batch_size = ::atoi(optarg); break;
```

Comment:
You can parse this into an `unsigned long int` (which is the same size as an `unsigned int`) using `strtoul(optarg, NULL, 10)`, which avoids the need for the cast below.